website/src/lib/oasst_api_client.ts

```diff
@@ -30,12 +24,66 @@ export class OasstApiClient {
     }
   }
 
-  fetch_full_settings() {
-    return this.get<Record<string, any>>("/api/v1/admin/backend_settings/full");
+  private async request<T>(
```

Comment:
this function has been moved to the top from below to reflect the call hierarchy in code